ui/main/src/app/business/services/processes.service.spec.ts

```diff
@@ -81,13 +80,22 @@ describe('Processes Services', () => {
         });
     });
 
-    describe('#queryProcess', () => {
-        const processDefinition = new Process('testPublisher', '0', 'businessconfig.label');
-        it('should load businessconfig from remote server', () => {
-            processServerMock.setProcessDefinition(processDefinition);
+    describe('Query a process', () => {
+        it('GIVEN a process exist WHEN query the process THEN process is return', () => {
```

Comment:
What do you think of 'GIVEN an existing process WHEN query the process THEN process is returned' ?